[Don't review yet, experimental] Use faster crc impl for opensource torch::save()

jjlilley opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32806 [Don't review yet, experimental] Use faster crc impl for opensource torch::save()**

Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

Commit:
[Don't review yet, experimental] Use faster crc impl for opensource torch::save()



Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 51551365:


* **11/11** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=51551365eae4f7041e8ba895dd840b88dfcba4f5).

### :detective: 11 new failures recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376574) binary_linux_manywheel_2_7mu_cpu_devtoolset7_build (1/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92955280) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92955280))
<details>
<summary>
<code>Jan 30 01:47:22 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:47:21 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:47:21 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:47:21 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:47:21 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:47:21 core header install: /pytorch/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:47:21 core header install: /pytorch/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:47:21 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:47:21 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:47:21 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:47:22 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:47:22 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:47:22   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:47:22  
Jan 30 01:47:22  
Jan 30 01:47:22 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:47:22 -- Including IDEEP operators 
Jan 30 01:47:22 -- Excluding image processing operators due to no opencv 
Jan 30 01:47:22 -- Excluding video processing operators due to no opencv 
Jan 30 01:47:22 -- MPI operators skipped due to no MPI support 
Jan 30 01:47:22 -- Include Observer library 
Jan 30 01:47:22 -- _GLIBCXX_USE_CXX11_ABI is already defined as a cmake variable 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376582) binary_linux_libtorch_2_7m_cpu_devtoolset7_shared-with-deps_build (2/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92955529) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92955529))
<details>
<summary>
<code>Jan 30 01:47:38 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:47:38 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:47:38 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:47:38 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:47:38 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:47:38 core header install: /pytorch/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:47:38 core header install: /pytorch/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:47:38 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:47:38 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:47:38 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:47:38 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:47:38 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:47:38   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:47:38  
Jan 30 01:47:38  
Jan 30 01:47:38 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:47:38 -- Including IDEEP operators 
Jan 30 01:47:38 -- Excluding image processing operators due to no opencv 
Jan 30 01:47:38 -- Excluding video processing operators due to no opencv 
Jan 30 01:47:38 -- MPI operators skipped due to no MPI support 
Jan 30 01:47:38 -- Include Observer library 
Jan 30 01:47:38 -- _GLIBCXX_USE_CXX11_ABI is already defined as a cmake variable 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376579) binary_linux_libtorch_2_7m_cpu_gcc5_4_cxx11-abi_shared-with-deps_build (3/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92955594) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92955594))
<details>
<summary>
<code>Jan 30 01:48:14 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:48:13 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:48:13 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:48:13 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:48:13 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:48:13 core header install: /pytorch/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:48:13 core header install: /pytorch/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:48:13 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:48:13 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:48:13 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:48:14 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:48:14 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:48:14   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:48:14  
Jan 30 01:48:14  
Jan 30 01:48:14 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:48:14 -- Including IDEEP operators 
Jan 30 01:48:14 -- Excluding image processing operators due to no opencv 
Jan 30 01:48:14 -- Excluding video processing operators due to no opencv 
Jan 30 01:48:14 -- MPI operators skipped due to no MPI support 
Jan 30 01:48:14 -- Include Observer library 
Jan 30 01:48:14 -- _GLIBCXX_USE_CXX11_ABI is already defined as a cmake variable 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376588) caffe2_onnx_py2_gcc5_ubuntu16_04_build (4/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92956375) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92956375))
<details>
<summary>
<code>Jan 30 01:48:53 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:48:53 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:48:53 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:48:53 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:48:53 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:48:53 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:48:53 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:48:53 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:48:53 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:48:53 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:48:53 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:48:53 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:48:53   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:48:53  
Jan 30 01:48:53  
Jan 30 01:48:53 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:48:53 -- Excluding ideep operators as we are not using ideep 
Jan 30 01:48:53 -- Including image processing operators 
Jan 30 01:48:53 -- Excluding video processing operators due to no opencv 
Jan 30 01:48:53 -- MPI operators skipped due to no MPI support 
Jan 30 01:48:53 -- Include Observer library 
Jan 30 01:48:53 -- /var/lib/jenkins/workspace/sccache/c++ /var/lib/jenkins/workspace/caffe2/../torch/abi-check.cpp -o /var/lib/jenkins/workspace/build/abi-check 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376573) caffe2_onnx_py3_6_clang7_ubuntu16_04_build (5/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92956747) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92956747))
<details>
<summary>
<code>Jan 30 01:49:13 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:49:12 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:49:12 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:49:12 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:49:12 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:49:12 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:49:12 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:49:13 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:49:13 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:49:13 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:49:13 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:49:13 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:49:13   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:49:13  
Jan 30 01:49:13  
Jan 30 01:49:13 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:49:13 -- Excluding ideep operators as we are not using ideep 
Jan 30 01:49:13 -- Including image processing operators 
Jan 30 01:49:13 -- Excluding video processing operators due to no opencv 
Jan 30 01:49:13 -- MPI operators skipped due to no MPI support 
Jan 30 01:49:13 -- Include Observer library 
Jan 30 01:49:13 -- Building c10d without CUDA/ROCm support 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376587) pytorch_linux_xenial_py2_7_9_build (6/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92957038) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92957038))
<details>
<summary>
<code>Jan 30 01:49:39 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:49:38    Host processor: x86_64 
Jan 30 01:49:38    Detected C compiler: GNU @ /opt/cache/bin/cc 
Jan 30 01:49:38 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:49:38 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:49:38 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:49:38 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:49:38 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:49:39 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:49:39 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:49:39 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:49:39 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:49:39   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:49:39  
Jan 30 01:49:39  
Jan 30 01:49:39 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:49:39 -- Including IDEEP operators 
Jan 30 01:49:39 -- Excluding image processing operators due to no opencv 
Jan 30 01:49:39 -- Excluding video processing operators due to no opencv 
Jan 30 01:49:39 -- MPI operators skipped due to no MPI support 
Jan 30 01:49:39 -- Include Observer library 
Jan 30 01:49:39 -- /opt/cache/bin/c++ /var/lib/jenkins/workspace/caffe2/../torch/abi-check.cpp -o /var/lib/jenkins/workspace/build/abi-check 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376571) pytorch_linux_xenial_py3_clang5_mobile_build (7/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92957353) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92957353))
<details>
<summary>
<code>Jan 30 01:50:00 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:50:00 -- Looking for shm_unlink - found 
Jan 30 01:50:00 -- Looking for malloc_usable_size 
Jan 30 01:50:00 -- Looking for malloc_usable_size - found 
Jan 30 01:50:00 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:50:00 core header install: /var/lib/jenkins/workspace/build_mobile/aten/src/ATen/core/TensorBody.h 
Jan 30 01:50:00 core header install: /var/lib/jenkins/workspace/build_mobile/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:50:00 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:50:00 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:50:00 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:50:00 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:50:00 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:50:00   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:50:00  
Jan 30 01:50:00  
Jan 30 01:50:00 -- /opt/cache/bin/c++ /var/lib/jenkins/workspace/caffe2/../torch/abi-check.cpp -o /var/lib/jenkins/workspace/build_mobile/abi-check 
Jan 30 01:50:00 -- Determined _GLIBCXX_USE_CXX11_ABI=1 
Jan 30 01:50:00 -- Using ATen parallel backend: NATIVE 
Jan 30 01:50:00 CMake Warning at CMakeLists.txt:596 (message): 
Jan 30 01:50:00   Generated cmake files are only fully tested if one builds with system glog, 
Jan 30 01:50:00   gflags, and protobuf.  Other settings may generate files that are not well 
Jan 30 01:50:00   tested. 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376589) pytorch_macos_10_13_py3_build (8/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92957544) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92957544))
<details>
<summary>
<code>Jan 29 17:50:06 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 29 17:50:05 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 29 17:50:05 -- RUNNING_ON_TRAVIS : 0 
Jan 29 17:50:05 -- COMPILER_SUPPORTS_OPENMP :  
Jan 29 17:50:05 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 29 17:50:05 core header install: /Users/distiller/project/build/aten/src/ATen/core/TensorBody.h 
Jan 29 17:50:05 core header install: /Users/distiller/project/build/aten/src/ATen/core/TensorMethods.h 
Jan 29 17:50:05 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 29 17:50:06 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 29 17:50:06 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 29 17:50:06 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 29 17:50:06 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 29 17:50:06   set_source_files_properties called with incorrect number of arguments. 
Jan 29 17:50:06  
Jan 29 17:50:06  
Jan 29 17:50:06 -- NCCL operators skipped due to no CUDA support 
Jan 29 17:50:06 -- Including IDEEP operators 
Jan 29 17:50:06 -- Excluding image processing operators due to no opencv 
Jan 29 17:50:06 -- Excluding video processing operators due to no opencv 
Jan 29 17:50:06 -- MPI operators skipped due to no MPI support 
Jan 29 17:50:06 -- Include Observer library 
Jan 29 17:50:07 -- Building c10d without CUDA/ROCm support 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376580) pytorch_linux_xenial_py3_6_gcc5_4_build (9/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92957780) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92957780))
<details>
<summary>
<code>Jan 30 01:50:10 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:50:10    Host system: Linux-4.15.0-1027-gcp 
Jan 30 01:50:10    Host processor: x86_64 
Jan 30 01:50:10    Detected C compiler: GNU @ /opt/cache/bin/cc 
Jan 30 01:50:10 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:50:10 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:50:10 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:50:10 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:50:10 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:50:10 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:50:10 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:50:10 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:50:10   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:50:10  
Jan 30 01:50:10  
Jan 30 01:50:10 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:50:10 -- Including IDEEP operators 
Jan 30 01:50:10 -- Excluding image processing operators due to no opencv 
Jan 30 01:50:10 -- Excluding video processing operators due to no opencv 
Jan 30 01:50:10 -- MPI operators skipped due to no MPI support 
Jan 30 01:50:10 -- Include Observer library 
Jan 30 01:50:10 -- /opt/cache/bin/c++ /var/lib/jenkins/workspace/caffe2/../torch/abi-check.cpp -o /var/lib/jenkins/workspace/build/abi-check 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376585) pytorch_linux_xenial_py3_clang5_asan_build (10/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92958762) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92958762))
<details>
<summary>
<code>Jan 30 01:50:35 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:50:35 -- SDE : SDE_COMMAND-NOTFOUND 
Jan 30 01:50:35 -- RUNNING_ON_TRAVIS : 0 
Jan 30 01:50:35 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:50:35 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:50:35 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:50:35 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:50:35 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:50:35 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:50:35 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:50:35 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:50:35 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:50:35   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:50:35  
Jan 30 01:50:35  
Jan 30 01:50:35 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:50:35 -- Excluding ideep operators as we are not using ideep 
Jan 30 01:50:35 -- Excluding image processing operators due to no opencv 
Jan 30 01:50:35 -- Excluding video processing operators due to no opencv 
Jan 30 01:50:35 -- MPI operators skipped due to no MPI support 
Jan 30 01:50:35 -- Include Observer library 
Jan 30 01:50:36 -- Building c10d without CUDA/ROCm support 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4376578) pytorch_xla_linux_xenial_py3_6_clang7_build (11/11)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92958945) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92958945))
<details>
<summary>
<code>Jan 30 01:50:44 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties):</code>
</summary>

```
Jan 30 01:50:43    Host processor: x86_64 
Jan 30 01:50:43    Detected C compiler: Clang @ /opt/cache/bin/clang 
Jan 30 01:50:43 -- COMPILER_SUPPORTS_OPENMP : 1 
Jan 30 01:50:43 AT_INSTALL_INCLUDE_DIR include/ATen/core 
Jan 30 01:50:43 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorBody.h 
Jan 30 01:50:43 core header install: /var/lib/jenkins/workspace/build/aten/src/ATen/core/TensorMethods.h 
Jan 30 01:50:43 -- Performing Test COMPILER_HAS_M_PCLMUL 
Jan 30 01:50:43 -- Performing Test COMPILER_HAS_M_PCLMUL - Success 
Jan 30 01:50:43 -- Performing Test COMPILER_HAS_M_SSE42 
Jan 30 01:50:44 -- Performing Test COMPILER_HAS_M_SSE42 - Success 
Jan 30 01:50:44 CMake Error at caffe2/serialize/CMakeLists.txt:11 (set_source_files_properties): 
Jan 30 01:50:44   set_source_files_properties called with incorrect number of arguments. 
Jan 30 01:50:44  
Jan 30 01:50:44  
Jan 30 01:50:44 -- NCCL operators skipped due to no CUDA support 
Jan 30 01:50:44 -- Including IDEEP operators 
Jan 30 01:50:44 -- Excluding image processing operators due to no opencv 
Jan 30 01:50:44 -- Excluding video processing operators due to no opencv 
Jan 30 01:50:44 -- MPI operators skipped due to no MPI support 
Jan 30 01:50:44 -- Include Observer library 
Jan 30 01:50:44 -- Building c10d without CUDA/ROCm support 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]

Commit:
Update on "[Don't review yet, experimental] Use faster crc impl for opensource torch::save()"


Differential Revision: [D19635487](https://our.internmc.facebook.com/intern/diff/D19635487/)

[ghstack-poisoned]
